src/main/java/org/commcare/formplayer/application/MenuController.java

```diff
@@ -61,6 +62,9 @@ public class MenuController extends AbstractBaseController {
     @Autowired
     private ResponseMetaDataTracker responseMetaDataTracker;
 
+    @Autowired
+    private FormplayerDatadog datadog;
```

Comment:
don't see it getting used in this class..

Reply:
ah i moved the code that used that to `MetricAspect`. Deleted here [5feea3c](https://github.com/dimagi/formplayer/pull/1551/commits/5feea3c80f5ed361b9add353d38d52757afd994a)